backend/packages/Upgrade/src/api/controllers/ExperimentController.ts

```diff
@@ -843,6 +843,7 @@ export class ExperimentController {
       } else {
         const policyParametersResponse = await this.moocletExperimentService.getPolicyParametersByExperimentId(id);
```

Comment:
You are making two queries to get the same mooclet experiment and extracting `policyParameter` from one and `rewardMetricKey` from another

Reply:
They both pull a reference to the same moocletExperimentRef that's true. But  `getPolicyParametersByExperimentId(id);` also fetches an updated copy of the policyParameters from the mooclet server.

Would you like me to combine these so we just make the one query to get moocletExperimentRef?

Reply:
@VivekFitkariwala I updated this to combine